bertly.py

```diff
@@ -170,3 +168,13 @@ def revoke(token):
         return jsonify({'success': 'hey nice job'}, 200)
     except RevokeError as e:
         return jsonify({'error': e}, 400)
+
+
+"""
+Error handlers
+"""
+
+@app.errorhandler(404)
+def page_not_found(e):
+    app.logger.warn("Bad URL: " + request.url)
+    return render_template('404.html'), 404
```

Comment:
Instead of rendering a template, it might be easier to just redirect to https://www.dosomething.org/404 or https://next.dosomething.org/404. No HTTP status, but less chance of layouts and stuff getting out of sync!